compactor/periodic_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/jonboulle/clockwork"
 )
 
-func TestPeriodic(t *testing.T) {
+func TestPeriodicHourly(t *testing.T) {
```

Comment:
we can just follow testing on this
https://github.com/coreos/etcd/blob/f65aee0759633966e0e1052d0c18079da551d24e/compactor/periodic_test.go#L27-L72

Reply:
we probably adjust `fc.Advance(checkCompactionInterval)` to `fc.Advance(6*time.Minute)` .